### PR TITLE
Clarify possible diabdat.mpq locations in README.md

### LINKED ISSUE #473: Request: Instructions for deploying on MacOS
It's mentioned in a number of places that the `diabdat.mpq` file should be placed in `~/.local/share/diasurgical/devilution` on Linux, but this doesn't work on MacOS. The following locations also don't work:

* `~/Library/diasurgical/devilution`
* `/Applications/devilutionx.app`
* `/Applications/devilutionx.app/diasurgical/devilution`
* `/Applications/devilutionx.app/Contents`
* `/Applications/devilutionx.app/Contents/diasurgical/devilution`
* `/Applications/devilutionx.app/Contents/MacOS`
* `/Applications/devilutionx.app/Contents/MacOS/diasurgical/devilution`
* `/Applications/devilutionx.app/Contents/Resources`
* `/Applications/devilutionx.app/Contents/Resources/diasurgical/devilution`

I've looked for instructions explaining where to deploy `diabdat.mpq` on MacOS in the wiki, readme and sources but haven't found anything. Something simple explaining how to get things running would be really helpful :)

Thanks!

### PR DESCRIPTION
**Pull request by MaxDesiatov**
Resolves #473

### CODE REVIEW

**Comment by AJenbo** [comment]
This isn't specific to macOS as:
Linux: ~/.local/share/diasurgical/devilution/
Windows: C:\Users\[username]\AppData\Roaming\diasurgical\devilution

But also the path may differ depending on OS version and security setting so giving the absolute path can be a bit tricky/misleading :(

**Comment by MaxDesiatov** [comment]
I've updated the PR accordingly. The main motivation is that #473 is not the first reported question about required location of `diabdat.mpq` as far as I'm aware, hopefully more detailed info in `README.md` will reduce the amount of issues created related to this.

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Status | Platform
 ![Discord Channel](https://avatars3.githubusercontent.com/u/1965106?s=16&v=4) [Discord Chat Channel](https://discord.gg/aQBQdDe)
 
 # How To Play:
- - Copy diabdat.mpq from your CD, or GoG install folder, to the DevilutionX install folder ; Make sure it is all lowercase.
+ - Copy diabdat.mpq from your CD, or GoG install folder, to the DevilutionX install folder; make sure it is all lowercase. On macOS placing the file in `~/Library/Application Support/diasurgical/devilution` directory would also work, same on Linux for `~/.local/share/diasurgical/devilution/` and on Windows for `C:\Users[username]\AppData\Roaming\diasurgical\devilution`. These path may differ depending on OS version and security setting, so the most reliable recommended location is the directory that contains DevilutionX executable.
```
Comment (AJenbo) [inline README.md]:
```suggestion
 - Copy diabdat.mpq from your CD, or GoG install folder, to the DevilutionX install folder; make sure it is all lowercase. On macOS placing the file in `~/Library/Application Support/diasurgical/devilution` directory would also work, same on Linux for `~/.local/share/diasurgical/devilution/` and on Windows for `C:\Users[username]\AppData\Roaming\diasurgical\devilution`. These path may differ depending on OS version and security setting, so the most reliable recommended location is the directory that contains the DevilutionX executable.
```

**Comment by AJenbo** [comment]
@MaxDesiatov did you see this one?
https://github.com/diasurgical/devilutionX/issues/501

**Comment by MaxDesiatov** [comment]
@AJenbo yes, the only other workaround I could mention is [the one with the context menu](https://support.apple.com/en-gb/guide/mac-help/mh40616/mac), but other than that we'd have to publish signed and notarized builds. This should remove the app from quarantine. I'm not sure what exactly is a virtual volume though in that report, a mounted DMG? Either way, do you think that this PR could be merged as is? The rest of the use cases could be clarified in separate PRs when more info is gathered.

**Comment by AJenbo** [comment]
Merged manually via https://github.com/diasurgical/devilutionX/commit/37b534e0420ae871439144a6da0ca9b2b63af3a4

**Pull request closed without merging**